testcases/OpTestEEH.py

```diff
@@ -261,6 +309,7 @@ def runTest(self):
                     break
                 else:
                     print "#Waiting for PHB %s recovery. (%d/%d)" % (domain,i,tries)
+            self.gather_logs()
```

Comment:
why this change? Gathering logs is fairly expensive. Can we instead gather them only on failure?